### DISCUSSION TITLE
Introduce yourself

### DISCUSSION THREAD

**Discussion by ibnesayeed** [cs531-f22/cs531-f22.github.io#1] [Announcements]
Hi,

I am [Dr. Sawood Alam](https://archive.org/~sawood/), a Web and Data Scientist of the [Wayback Machine](https://web.archive.org/) at the [Internet Archive](https://archive.org/). I will be the instructor for the CS 431/531 Web Server Design course this Fall 2022 semester. Looking forward to seeing you all in the class.

**Comment by darinmclain** [comment]
Hello,

I am Darin McLain, CS Masters with ICT concentration student.   I graduated from ODU in 2009 with a BSBA in Information Systems (IT) and have been working in the industry since then.  I've worked as a software developer/engineer my entire career, working at DoD contractors, Sentara, Disney and now at CarMax.  

I started my career in the LAMP stack, but have been Microsoft focused since around 2012,  and have been a full stack engineer the whole time.  

This should be my final year to get my Masters and I'm really looking forward to the class!

**Comment by calebkbrad** [comment]
Hey all.
I am Caleb Bradford and I am currently an undergraduate senior taking this class at the graduate level through the Linked Program. I transferred to ODU last fall after earning my Associate's at TCC. Over the summer, I was a participant in the first cohort of the ODU CS Undergraduate Research Experience in Disinformation Detection and Analysis, where I worked on a research project with Dr. Michael Nelson. 
I'm looking forward to learning more about HTTP through this class!

**Comment by rohitrahiya** [comment]
Hi everyone,
I, am Rohit Chaudhary, came to ODU in Spring 22 to pursue my masters in computer Science. I've done my Bachelors in CS and have worked as a network engineer for 7.5yrs which gave me wisdom and strength to start my learning from scratch to help me become a DS/ML engineer someday. I have no prior experience of programing and have never done any significant research or project but I have a can do attitude that got me here which is also my expectation from this course that would eventually help gain better understanding and a new perspective of web mechanism. 

Thank you all and I'm really glad that we all are here together. Have a wonderful Web Server Design..

**Comment by Badjedi04** [comment]
Hello,
I am Prashant Tomar and I am a Graduate student came to ODU in Fall-2022 to pursue my Masters in Computer Science. I've done my Undergrad in Computer Science and have the experience of 7.5 years of working as a Network Engineer. I'm expecting to learn the concepts of HTTP through this class.

**Comment by Licentiathe8th** [comment]
My name is Richard Richmond and I am an undergraduate student at ODU earning a bachelor degree in Electrical Engineering Technology - Computer Engineering Technology Concentration.  It has been years since I have taken a course in Computer Science and I hope to regain some of the knowledge I've forgotten since then by taking this course.

**Comment by JMNewsome** [comment]
Hello everyone,

My name is Jozef Newsome, I am currently an undergraduate student at ODU. I am working on completing my bachelors in Computer Science. I love making programs and started building my first games in middle school. I am mostly self taught, but have broadened my knowledge considerably by working on my degree. Web server design is an interest because I am working on an app for iOS and Android that involves using REST APIs to communicate. My goal is to have a more rounded and thorough understanding of server design to created a better product and apply the knowledge to my career.

**Comment by yaswanth9909** [comment]
Hello everyone,
My name is Yaswanth Bondalapati,I am currently a Masters student in Odu. I am pursuing Masters in Computer Science and this is my second semester as a graduate student. I did my undergrad in the Vellore Institute of Technology(VIT),India .I have no prior knowledge of this course and I'm hoping to learn everything about building a webserver.
